src/content/docs/how-to/local.mdx

```diff
@@ -42,17 +51,18 @@ bluebuild build ./recipes/recipe.yml
 
 You can see that the BlueBuild GitHub Action [uses the same `build` command][bluebuild-cli-ghaction-build], with additional `--push` and `--registry` options for pushing successful build to `ghcr.io` or similar image repositories.
 
-You can see the complete list of available options using `bluebuild build help`.
+You can see the complete list of available options using `bluebuild build --help`.
 
-### Upgrade
+### Rebase/Upgrade
 
 :::caution
-This section is a draft, so contents are unfinished or nonexistent. The CLI interface may be refactored in the future; see [here][refactor-discussion] for more information.
+These commands will be replace by `switch` in `v0.9.0`. It will be aliased with `rebase` and `upgrade` so as to not break existing scripts. Right now, `upgrade` must be called when building the same image you called `rebase` on earlier. The `switch` command removes this ambiguity.
```

Comment:
```suggestion
These commands will be replaced by `switch` in `v0.9.0`. It will be aliased with `rebase` and `upgrade` so as to not break existing scripts. Right now, `upgrade` must be called when building the same image you called `rebase` on earlier. The `switch` command removes this ambiguity.
```